dev-tools/iota-private-network/experiments/dual-run.sh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2025 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+# dual-run.sh — run sets of experiments (Mysticeti + Starfish per step)
+set -euo pipefail
+IFS=$'\n\t'
+
+
+# ---------------------------------------------------------------------
+# Paths
+# ---------------------------------------------------------------------
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+DOCKER_ROOT="$(cd "$SCRIPT_DIR/../../../docker" && pwd)"
+
+echo "dual-run: script dir = $SCRIPT_DIR"
+echo "dual-run: docker root = $DOCKER_ROOT"
+
+
+# ---------------------------------------------------------------------
+# Build images ONCE up front
+# ---------------------------------------------------------------------
+echo "dual-run: building docker images once (node, tools, indexer)..."
+
+# Use sudo because your builds / docker usually run as root
+if ! (cd "$DOCKER_ROOT/iota-node"    && sudo ./build.sh -t iota-node); then
```

Comment:
```suggestion
if ! (cd "$DOCKER_ROOT/iota-node"    && sudo ./build.sh); then
```
`build.sh` scripts do not take any arguments.